harvdev_utils/production/production.py

```diff
@@ -4635,8 +4635,8 @@ class Phendesc(Base):
 
     def __str__(self):
         """Over write the default output."""
-        return "Phendesc id = {}\n\tgenotype:'{}'\n\tenvironment:'{}'\n\ttype{}\n\tpub:{}".\
-            format(self.phendesc_id, self.genotype, self.environment, self.type, self.pub)
+        return "Phendesc id = {}\n\tgenotype:'{}'\n\tenvironment:'{}'\n\ttype{}\n\tpub:{}\n\tdesc:{}...{}".\
+            format(self.phendesc_id, self.genotype, self.environment, self.type, self.pub, self.description[:10], self.description[:-10])
```

Comment:
I think you're trying to print out the 1st 10 chars of the phendesc.description, ellipsis, and then the last 10 chars of phendesc.description. If so, then I think you need to change the last bit from `[:-10]` to `[-10:]`.

Reply:
bizarrely though it worked. Have changed it now anyway, thanks.